integration/integration_test_utils.py

```diff
@@ -294,3 +324,32 @@ def _convert_nginx_to_python_type(typ, value):
         return datetime.utcfromtimestamp(secs + nanosecs / 1000000000)
     else:
         raise V3ioError(f"Type {typ} in get item response is not supported")
+
+
+def create_sql_table(schema, table_name, sql_db_path, key):
+    import datetime
+
+    import pandas as pd
+    import sqlalchemy as db
```

Comment:
These should be imported at the top of the file.
```suggestion
    import sqlalchemy as db
```